### PR TITLE
fix: allow version pass to avoid latest install always

### LINKED ISSUE #8513: fix: startjanssendemo script always install latest
wget https://raw.githubusercontent.com/GluuFederation/flex/v5.1.0/automation/startflexdemo.sh installs 5.1.1

### PR DESCRIPTION
**Pull request by misba7**
closes #8513

### CODE REVIEW

File: automation/startjanssendemo.sh [hunk 1]
```diff
@@ -2,10 +2,10 @@
 set -eo pipefail
 JANS_FQDN=$1
 JANS_PERSISTENCE=$2
-JANS_CI_CD_RUN=$3
+JANS_VERSION=$3
```
Comment (iromli) [inline automation/startjanssendemo.sh:5]: @misba7 What's the default `JANS_VERSION`?

File: automation/startjanssendemo.sh [hunk 2]
```diff
@@ -242,7 +239,7 @@ auth-server:
 EOF
 sudo helm repo add janssen https://docs.jans.io/charts
 sudo helm repo update
-sudo helm install janssen janssen/janssen -n jans -f override.yaml --kubeconfig="$KUBECONFIG" "$HELM_DEVELOPMENT_REPO"
+sudo helm install janssen janssen/janssen -n jans -f override.yaml --kubeconfig="$KUBECONFIG" --version="$JANS_VERSION"
```
Comment (iromli) [inline automation/startjanssendemo.sh:242]: @misba7 have we handle empty `$JANS_VERSION`?

Comment (misba7) [reply automation/startjanssendemo.sh:242]: empty JANS_VERSION installs latest, which was the default behavior.
I added the variable to be able to override it as needed.

Comment (iromli) [reply automation/startjanssendemo.sh:242]: did you mean command like `helm install --version=` (empty JANS_VERSION) also work?

Comment (iromli) [reply automation/startjanssendemo.sh:242]: ignore my last comment, i have tested it. The `--version= <other-ops>` is working while `--version <other-ops>` is not.

**Review by iromli — APPROVED** [review]

**Review by moabu — APPROVED** [review]

**Pull request merged**